sources/light_client.move

```diff
@@ -1,23 +1,26 @@
 module bitcoin_spv::light_client;
 
-use bitcoin_spv::block_header::new_block_header;
+use bitcoin_spv::block_header::{BlockHeader, new_block_header};
 use bitcoin_spv::light_block::{LightBlock, new_light_block};
 use bitcoin_spv::merkle_tree::verify_merkle_proof;
 use bitcoin_spv::btc_math::target_to_bits;
 use bitcoin_spv::utils::nth_element;
 
-use sui::dynamic_object_field as dof;
+use sui::dynamic_field as df;
 
 const EBlockHashNotMatch: u64 = 1;
 const EDifficultyNotMatch: u64 = 2;
 const ETimeTooOld: u64 = 3;
+const EHeaderListIsEmpty: u64 = 4;
+const EBlockDoesnotExist: u64 = 5;
+const EForkNotEnoughPower: u64 = 6;
```

Comment:
power or difficulty? Or "below target"?

Reply:
This is the chain_work. EChainWorkLessPower may make more sense. Would you happen to have any suggestions?

Reply:
let's do: `EForkChainWorkTooSmall`